app/javascript/pages/account/my-account.tsx

```diff
@@ -62,7 +63,8 @@ const MyAccount = (_props: MyAccountProps) => {
   const { profile, loading, initialStateLoaded } = React.useContext(UserContext)
 
   if (!profile && !loading && initialStateLoaded) {
-    // TODO: Redirect to React sign in page and show a message that user needs to sign in
+    window.sessionStorage.setItem("redirect", "account")
+    setSiteAlertMessage(t("signIn.loginRequired"), "secondary")
```

Comment:
thought: We're well along this path now, but in the future we should look into using constants instead of strings for things like "redirect", "account", and "secondary".